web/app/controllers/application_controller.rb

```diff
@@ -2,4 +2,9 @@ class ApplicationController < ActionController::Base
   # Prevent CSRF attacks by raising an exception.
   # For APIs, you may want to use :null_session instead.
   protect_from_forgery with: :exception
+
+  rescue_from CanCan::AccessDenied do |exception|
+    flash[:error] = exception.message
```

Comment:
Yeah so there's 3 possible arguments for flash, :error, :notice, or :alert.  the next page navigated to will get this flash passed to it, and it will only be displayed once.  So if the user refreshes the flash will disappear.  Depending on the type the message will display differently.  If you use :notice, it will display the message on the top and bottom of the screen.  :error just displays it once where you put it in the html.  Go to the bloggs/index.html.erb and see 
+<% if flash[:error] %>
+    <p class="notice"><%= flash[:error] %></p>
+<% end %>

Which just checks if a flash was passed to the page, and if so then we just flash the error